bundles/ui/org.eclipse.smarthome.ui.icon/src/main/java/org/eclipse/smarthome/ui/icon/internal/IconServlet.java

```diff
@@ -41,6 +45,7 @@
  *
  * @author Kai Kreuzer - Initial contribution
  */
+@Component
```

Comment:
The original service xml stated also `configuration-policy="optional"` and `modified="modified"`.